SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -188,6 +188,26 @@
 
   private int maxTasksPerOfferPerRequest = 0;
 
+  private double minOfferScore = 0.80;
+
+  private int maxOfferAttemptsPerTask = 20;
+
+  private long maxMillisPastDuePerTask = TimeUnit.MINUTES.toMillis(10);
```

Comment:
thinking something shorter would be a better default. We can check in testing, but I think after even 5 minutes we shouldn't be worrying about score and just scheduling asap